builds/e2e/proxy-deployment-template.json

```diff
@@ -0,0 +1,396 @@
+{
```

Comment:
Nice! This is an ARM template to set up the VMs?

Reply:
Just want to verify this is scrubbed for any sensitive information, subscription ids, etc.

Reply:
Yes, it's an ARM template. The sensitive info would get passed in as parameters to this template from the bash script, and I elided the subscription ID, etc there as `<>`